OpenRA.Mods.Common/Activities/FindAndDeliverResources.cs

```diff
@@ -239,17 +224,19 @@ public override IEnumerable<TargetLineNode> TargetLineNodes(Actor self)
 
 			if (orderLocation != null)
 				yield return new TargetLineNode(Target.FromCell(self.World, orderLocation.Value), harvInfo.HarvestLineColor);
-			else if (deliverActor != null)
-				yield return new TargetLineNode(Target.FromActor(deliverActor), harvInfo.DeliverLineColor);
+			else
+			{
+				var manager = harv.DockClientManager;
+				if (manager.ReservedHostActor != null)
+					yield return new TargetLineNode(Target.FromActor(manager.ReservedHostActor), manager.DockLineColor);
+			}
 		}
 
-		CPos? GetSearchFromProcLocation()
+		CPos? GetSearchFromProcLocation(Actor self)
```

Comment:
This function is a bit cryptically named, it should include at least a one line comment explaining what it does above it.

Reply:
I suppose that function is now so short that I can just inline the code

Reply:
Inlined and added an extra comment

Reply:
Yep looks good now.